### ISSUE TITLE
Possible to view terraform stdout / stderr while running acceptance tests

### ISSUE THREAD

**Issue by wyardley** [hashicorp/terraform-provider-google#20049]
### Question

Feel free to convert this to an enhancement in the appropriate category if appropriate, or give me a pointer to the right flags  / arguments if there's a way to do this already.

While running tests locally, it would be very convenient to be able to view the stdout / stderr from the terraform calls.

While using `TF_LOG=debug` (or `trace`) can be helpful, there's still sometimes missing information about what's going on (especially since "Step 4/7" doesn't always make it 100% clear which config it's on or what the plan looked like). This is especially true for working on things like the container service where there are a ton of loops waiting on things to update / converge.

Being able to see the plans / apply / etc. output, and / or to otherwise view more detailed information about the specific parameters being applied at a given time would make troubleshooting and writing tests easier.

**Comment by ggtisc** [comment]
Hi @wyardley 

To open and enhancement issue you just need to select that category when you are opening the issue.

**Comment by wyardley** [comment]
> To open and enhancement issue you just need to select that category when you are opening the issue.

@ggtisc yes, I know that, I just wasn't sure if this qualified as a bug or an enhancement; the comment was just that you all could recategorize it if you felt it belonged to a different category. I'll reopen as an enhancement.
